lib/instrumentation/transaction.js

```diff
@@ -133,7 +139,8 @@ Transaction.prototype.toJSON = function () {
     span_count: {
       started: this._builtSpans
     },
-    outcome: this.outcome
+    outcome: this.outcome,
+    faas: this._faas
```

Comment:
It would be nice to either have the `this._faas` initial value be `undefined` or only add `faas` to `payload` behind a `if (this._faas)` guard. Otherwise, in the common non-faas case we will be adding `"faas": null` to serialized transaction APM events.

Reply:
✅ in 60b6f4a166422407f3b9c78e88ad211cc3c4b23c